tests/post_training/pipelines/masked_language_modeling.py

```diff
@@ -57,17 +62,18 @@ def _dump_model_fp32(self) -> None:
             ov_model = convert_model(onnx_path)
             ov.serialize(ov_model, self.output_model_dir / "model_fp32.xml")
 
-        if self.backend in OV_BACKENDS:
+        if self.backend in OV_BACKENDS + [BackendType.FP32]:
```

Comment:
Minor. It might be more useful to create `ALL_BACKENDS = OV_BACKENDS + [BackendType.FP32]` instead of pushing addition everywhere.